requirements.txt

```diff
@@ -1,3 +1,3 @@
 pymongo==2.7.1
 wheel==0.23.0
-
+numpy==1.9
```

Comment:
What I meant was that numpy is a very large library, having many moving parts, of which we are only using one very small portion.  And it's compiled.  And anything that we add as a dependency to dborutils has just become a dependency for all of the rest of our projects (well, dbor and noodleapi, I'm less familiar with others).